src/bootstrap.tsx

```diff
@@ -286,6 +286,20 @@ const setupSHOGunMap = async (application: Application) => {
 
   view.setConstrainResolution(true);
 
+  const processLayerGroup = new OlLayerGroup({
```

Comment:
What is a `processLayer`? To me this seems to be project related? Why do we need to initialize it in the shogun-gis-client?